src/lib/utils.js

```diff
@@ -123,3 +123,7 @@ export function feature (key, user, variation) {
     });
   });
 }
+
+export function reset() {
```

Comment:
Because this exists purely for testing, it would be good to make that explicit to resist accidental reliance.  e.g.
```
export const testExports = {
  reset: () => { ldClient = null; }
};
```